frontend/app/scripts/modules/serviceCenter/controllers/schemaCtrl.js

```diff
@@ -146,7 +146,7 @@ angular.module('serviceCenter.sc')
                 httpService.apiRequest(url, method, null, headers, "nopopup").then(function(response) {
                     if (response && response.data && response.data.schema) {
                         $scope.template = response.data.schema;
-                        $scope.template = $scope.template.replace(/\\/g, "");
+                        $scope.template = $scope.template.replace(/\\\s/g, "");
```

Comment:
it will replace the space to empty now. The description of schema maybe changed to unreadable.

Reply:
This will replace the '\' character with nothing only when it is the last character in the line :  

For Example : 
```
description: "body返回： <br/>{\"body\":{},\"statusCode\":\"120\",\"errorMsg\" \
       :\"没有为此用户分配有效的坐席\"}"
```
In this , it will just replace the last '\' in the first line, 
after replacing it becomes
```
description: "body返回： <br/>{\"body\":{},\"statusCode\":\"120\",\"errorMsg\" :\"没有为此用户分配有效的坐席\"}"
```